flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/PendingCheckpoint.java

```diff
@@ -602,6 +624,17 @@ private void reportFailedCheckpoint(Exception cause) {
 		}
 	}
 
+	private boolean grabStatusPriority() {
```

Comment:
Could you think of a more straightforward name? It's a kind of releasing mutex here, "priority" is not accurate.